test/get-config.test.js

```diff
@@ -1,12 +1,15 @@
-const path = require('path');
-const {format} = require('util');
-const test = require('ava');
-const {writeFile, outputJson} = require('fs-extra');
-const {omit} = require('lodash');
-const td = require('testdouble');
-const {stub} = require('sinon');
-const yaml = require('js-yaml');
-const {gitRepo, gitTagVersion, gitCommits, gitShallowClone, gitAddConfig} = require('./helpers/git-utils');
+import path from 'node:path';
+import {format} from 'node:util';
+import test from 'ava';
+import fsExtra from 'fs-extra';
+import {omit} from 'lodash-es';
+import * as td from 'testdouble';
+import yaml from 'js-yaml';
+import {gitAddConfig, gitCommits, gitRepo, gitShallowClone, gitTagVersion} from './helpers/git-utils.js';
+
+const {outputJson, writeFile} = fsExtra;
+const pluginsConfig = {foo: 'bar', baz: 'qux'};
```

Comment:
i'm interested in thoughts on meaningless data to be returned from a stub. it is meaningless here because the actual value of the data does not matter in any of the uses. it only matters that what is returned from the stub matches the portion of the return value of the called function that is expected. 

i normally use a random data generator, like https://github.com/travi/any, for cases like this so that it is clear that the data is meaningless. i find that it also makes it more clear when certain bits of data _are_ important for the behavior of the test, such as the specific attribute that causes a test to execute a particular conditional path. however, doing that here would be inconsistent with the rest of the project (and i want to avoid broader changes like using such an approach somewhat consistently).

i'm interested in opinions on techniques that make sense in this project for meaningless data like this. if longer term thoughts are different than short term thoughts, i'm interested in those as well, even though we'd avoid big changes at this point.

Reply:
I'd avoid using `foo`/`bar`, I'd use something more descriptive. 

I see the point for using random data, but I'm not sure if the added complexity is worth it? I'd read the code and wonder why we are setting mock data randomly? But you clearly thought more about it than I did, do what you think is best